ethcore/src/client/client.rs

```diff
@@ -771,7 +771,7 @@ impl<V> BlockChainClient for Client<V> where V: Verifier {
 			nonce: self.latest_nonce(a),
 			balance: self.latest_balance(a),
 		};
-		self.miner.import_transactions(transactions, fetch_account)
+		self.miner.import_transactions(self, transactions, fetch_account)
```

Comment:
This function (`BlockChainClient::import_transactions`) doesn't appear to be used anywhere - we only use `import_queued_transactions` now.